src/filtrering/filtrering-veileder-grupper/ny_veiledergrupper-innhold.tsx

```diff
@@ -1,66 +1,43 @@
 import {AppState} from '../../reducer';
-import {harGjortEndringer, veilederlisterErLik} from '../../components/modal/veiledergruppe/veileder-gruppe-utils';
+import {harGjortEndringer} from '../../components/modal/veiledergruppe/veileder-gruppe-utils';
 import {VeilederGruppeModal} from '../../components/modal/veiledergruppe/veileder-gruppe-modal';
 import {FiltervalgModell} from '../../model-interfaces';
 import {useEnhetSelector} from '../../hooks/redux/use-enhet-selector';
 import {visIngenEndringerToast} from '../../store/toast/actions';
-import {logEvent} from '../../utils/frontend-logger';
-import {finnSideNavn} from '../../middleware/metrics-middleware';
 import '../../components/sidebar/sidebar.less'
 import './ny_veileder-gruppe.less'
 import {ThunkDispatch} from "redux-thunk";
 import {AnyAction} from "redux";
 import {ListevisningType} from "../../ducks/ui/listevisning";
+import {LagretFilter} from "../../ducks/lagretFilter";
+import VeilederGruppeRad from "./ny_veileder_gruppe_rad";
 
 
 interface VeilederGruppeInnholdProps {
-    lagretFilter: VeiledergrupperFilter[]
+    lagretFilter: LagretFilter[]
     filterValg?: FiltervalgModell;
-    filtergruppe: string;
+    filtergruppe: ListevisningType;
 }
 
 function isOverflown(element) {
     return element.scrollHeight > element.clientHeight || element.scrollWidth > element.clientWidth;
 }
 
 function NyVeilederGruppeInnhold(props: VeilederGruppeInnholdProps) {
-    const [valgtGruppe, setValgtGruppe] = useState<VeiledergrupperFilter>();
     const [visEndreGruppeModal, setVisEndreGruppeModal] = useState(false);
-
-    const filtreringVeilederoversikt = (state: AppState) => state.filtreringVeilederoversikt.veiledere;
-    const filtreringEnhetensoversikt = (state: AppState) => state.filtreringEnhetensOversikt.veiledere;
-    const selector = props.filtergruppe === ListevisningType.enhetensOversikt ? filtreringEnhetensoversikt : filtreringVeilederoversikt;
-
-    const veiledereFilter = useSelector(selector);
-
-    useEffect(() => {
-        const valgtFilter = props.lagretFilter.find(v => veilederlisterErLik(v.filterValg.veiledere, veiledereFilter));
-        if (valgtFilter) {
-            setValgtGruppe(valgtFilter);
-        }
-    }, [veiledereFilter, valgtGruppe, props.lagretFilter]);
+    const valgtGruppeEngetensOversikt = useSelector((state: AppState) => state.mineFilterEnhetensOversikt.valgtVeilederGruppe);
+    const valgtGruppeVeilederOversikt = useSelector((state: AppState) => state.mineFilterVeilederOversikt.valgtVeilederGruppe);
+    const valgtGruppe = (props.filtergruppe === ListevisningType.veilederOversikt ? valgtGruppeVeilederOversikt : valgtGruppeEngetensOversikt)
 
     const outerDivRef = useRef<HTMLDivElement>(null);
 
     const dispatch: ThunkDispatch<AppState, any, AnyAction> = useDispatch();
     const enhet = useEnhetSelector();
 
-    const velgGruppe = (gruppeId: string) => {
-        logEvent('portefolje.metrikker.veiledergrupper.velg-gruppe',
```

Comment:
where is the logEvent for this now?

Reply:
https://github.com/navikt/veilarbportefoljeflatefs/blob/e7e5ca971f303ed3fbb74662b030c931cb71bae1/src/filtrering/filtrering-veileder-grupper/ny_veileder_gruppe_rad.tsx#L25